File: SecondYear/jeezzzz_AjeeshRawal_2226cseai1021_2/Week_1/pattern7.java
class Solution {

    void printDiamond(int n) {
       int space=n-1,i,j,k;
       for(i=1;i<=n;i++)
       {
           for(k=1;k<=space;k++)
           {
               System.out.print(" ");
           }
           space--;
           for(j=1;j<=i;j++)
           {
               System.out.print("* ");
           }
           System.out.println();
       }
       space=0;
       for(i=n;i>=1;i--)
       {
           for(k=1;k<=space;k++)
           {
               System.out.print(" ");
           }
           space++;
           for(j=1;j<=i;j++)
           {
               System.out.print("* ");
           }
           System.out.println();
       }
       
    }
}

File: SecondYear/swarnkarvikash_VikashKumarSwarnkar_2226csit1034_2/Week_1/pattern7.cpp
class Solution {
  public:
    void printDiamond(int n) {
        // code here
        for(int i=0;i<n;i++)
        {
            for(int space=0;space<n-i-1;space++)
            {
                cout<<" ";
            }
            for(int j=0;j<i+1;j++)
            {
                cout<<"*"<<" ";
            }
            cout<<endl;
        }
        for(int i=0;i<n;i++)
        {
            for(int space=0;space<i;space++)
            {
                cout<<" ";
            }
            for(int j=n-i;j>0;j--)
            {
                cout<<"*"<<" ";
            }
            cout<<endl;
        }
    }
};